lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,96 @@
-# Detailed Reference
+# Detailed Reference Guide
 
-?> Coming Soon
+!> Under Construction.
+
+## Platform Setup
+
+Before implementing the Lotame Lightning Tag on your site, there is setup needed to be completed inside the Lotame Platform as described in the [Pre-Implementation Setup Tasks](lightning-tag/implementation-setup-tasks.md)
+
+## Javascript Setup
+
+Setup of the Lotame Lightning Tag javascript on your website involves Passing a input object to a function which loads the window namespace on your page which is where the javascript tag loads itself into an executes. The basic setup looks like the below:
+
+```javascript
+<script>
+     var lotameTagInput = {
+         data: {},
+         config: {
+             clientId: CLIENT_ID
+         }
+     };
+
+     ! function(input) {
+         input = input || {};
+         var config = input.config || {};
+         namespace = window['lotame_' + config.clientId] = {};
```

Comment:
is it lotame_<id>?  I know for ccjs it was lotcc_, and it was changed but i thought it was changed to something with lt in it?  or is that not what this is supposed to be?

Reply:
@markaconrad Can you confirm it is `lotame_`?

Reply:
Yes, it is lotame_

Reply:
Thanks!

Reply:
ok, sorry for the confusion!